### ISSUE TITLE
[BUG] When applying prone, unconscious or death to a token Warning pop up only for players

### ISSUE THREAD

**Issue by Dynsai** [Larkinabout/fvtt-custom-dnd5e#91]
**Describe the bug**

![image](https://github.com/user-attachments/assets/f08fc0d2-85e8-400d-b7d7-74ac7af5f11c)
When I apply to a token the status prone, death or unconscious and this warning pop up. This occurs to any token the player doesn't own.
There are 3 warnings pop ups. If i remove the values on "Rotation" and "Tint" on Dead category and Prone category on the settings configuration of the module the pop up warnings are reduced to 1 instead of 3.

**Module Data**

Go into the module's settings, open up the Debug Settings menu, click Export Data, save and attach the file here.
[custom-dnd5e.json](https://github.com/user-attachments/files/16749934/custom-dnd5e.json)

**Steps to reproduce**
1. Go to a token
2. Click on death, prone or unconscious status
3. Players will se a pop up warning

**Versions:**
- Module: 0.14.0
- System: dnd5e 3.3.1
- Foundry VTT: Tried on 12.328 and 12.331
- Browser: Google Chrome 127.0.6533.120 and Firefox 129.0.2

**Additional context**
When i start the world there is an alert about incompatibility with monk's Bloodsplats. I disable it but the error persist.

Example of the warning on the console:
```
User Y - Raúl lacks permission to update Token [UQcPMFOt3Jia3R79] in parent Scene [zxDGecQ8sZD9TBe5]
fetch	@	foundry.js:71791
_remove	@	foundry.js:71775
(anonymous)	@	foundry.js:71794
setTimeout		
fetch	@	foundry.js:71794
notify	@	foundry.js:71687
🎁call_wrapped	@	libWrapper-wrapper.js:507
_lw_ui_notifications_notify	@	Bundle.js:132169
🎁call_wrapper	@	libWrapper-wrapper.js:616
🎁ui.notifications.notify#0	@	libWrapper-wrapper.js:189
error	@	foundry.js:71724
#handleError	@	foundry.js:3454
(anonymous)	@	foundry.js:3437
value	@	socket.js:580
value	@	socket.js:499
U.emit	@	index.mjs:136
(anonymous)	@	manager.js:204
Promise.then		
ce	@	websocket-constructor.browser.js:5
value	@	manager.js:203
U.emit	@	index.mjs:136
value	@	index.js:146
value	@	manager.js:190
U.emit	@	index.mjs:136
value	@	socket.js:341
U.emit	@	index.mjs:136
value	@	transport.js:98
value	@	transport.js:90
ws.onmessage	@	websocket.js:68
```

I tried a lot of things (4 hours trying), other modules, libwrapper settings, checking the code (i have very limited knowledge of js) but i couldn't find anything. Any help is appreciate. Thanks!

**Comment by Larkinabout** [comment]
Does the token still rotate/tint? I expect I just need to check that the user trying to apply the properties are the GM or owner.

**Comment by Bobster96Lobster** [comment]
Hello having the same issue here. It doesn't break anything but is annoying enough to turn off the module for my players.

**Comment by Larkinabout** [comment]
Fixed in 0.15.0.
